scriptmodules/emulators/mupen64plus/mupen64plus.sh

```diff
@@ -436,12 +436,20 @@ getAutoConf mupen64plus_texture_packs && useTexturePacks
 
 if [[ "$(sed -n '/^Hardware/s/^.*: \(.*\)/\1/p' < /proc/cpuinfo)" == BCM* ]]; then
     WINDOW_MODE="--windowed $RES"
-    SDL_VIDEO_RPI_SCALE_MODE=1
-    # If a raspberry pi is used lower resolution to 320x240 and enable SDL dispmanx scaling mode 1
+    if isPlatform "videocore"; then
```

Comment:
I don't think you can use `isPlatform` here because this script does not run under RetroPie's scripting environment.
You probably will need to copy the detection logic from the `system.sh` library to re-use here.

https://github.com/RetroPie/RetroPie-Setup/blob/master/scriptmodules/system.sh#L291-L296

I know is not very elegant but `system.sh` is not meant to be imported by 3rd-party scripts afaik.

Reply:
yup, i forgot. i put that stuff behind the iniGet instead. the ini option isn't set unless it's videocore